plugin-packages/ktx2/src/ktx2-container.ts

```
@@ -0,0 +1,274 @@
+    const kvdReader = new BufferReader(data, kvdByteOffset, kvdByteLength, true);
+
+    while (kvdReader.position < kvdByteLength) {
+      const keyValueByteLength = kvdReader.nextUint32();
+      const keyData = kvdReader.scan(keyValueByteLength);
+      const key = decodeText(keyData);
+
+      // 4-byte alignment.
+      const valueData = kvdReader.nextUint8Array(keyValueByteLength - keyData.byteLength - 1);
+
+      // eslint-disable-next-line no-control-regex
+      this.keyValue[key] = key.match(/^ktx/i) ? decodeText(valueData).replace(/^(.*)\x00$/, '$1') : valueData;
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)